kateryna_kravchuk/task_03.md

```
@@ -0,0 +1,243 @@
+
+    #1) Respect the privacy of others.
+    #2) Think before you type.
+    #3) With great power comes great responsibility.
+
+[sudo] password for kate_kr: 
+```  
+Note: I should confirm any action I want to do in the root directory by providing a root password.
+
+---
+
+#### 8. Change the owner of the copied file to root and adjust its access rights to 644 to restrict permissions appropriately.
```

markdownlint-cli2 (0.17.2)

Trailing punctuation in heading
Punctuation: '.'

(MD026, no-trailing-punctuation)